src/TelegramBot/TelegramBot.js

```diff
@@ -0,0 +1,71 @@
+import NodeTelegramBot from 'node-telegram-bot-api';
+
+export class TelegramBot {
+    constructor(token, options) {
+        this.bot = new NodeTelegramBot(token, options);
+    }
+
+    on(event, listener) {
+        return this.bot.on(event, listener);
+    }
+
+    emit(type, message, metadata) {
+        return this.bot.emit(type, message, metadata);
+    }
+
+    removeListener(eventName, listener) {
+        return this.bot.removeListener(eventName, listener);
+    }
+
+    sendMessage(nodeName, chatId, data) {
+        switch (nodeName) {
+            case 'text': {
+                return this.bot.sendMessage(chatId, data.text, data);
+            }
+            case 'img': {
+                return this.bot.sendPhoto(chatId, data.src, data);
+            }
+            default: {
+                throw new Error('tag ' + nodeName + ' does not exist');
```

Comment:
'Tag ${nodeName } does not exist.'

Reply:
'Tag ${nodeName } does not exist. ...Instructions on what to do...... '

Reply:
Added